doc/en/components/charts/chart-type-overview.md

```diff
@@ -1,32 +1,212 @@
+
+Learn more about our [pie charts](../pie-chart.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Line Chart
+
+The $PlatformShort$ Line Chart, or Line Graph is a type of category line graph shows the continuous data values represented by points connected by straight line segments of one or more quantities over a period time for showing trends and performing comparative analysis. The Y-Axis (labels on left side) show a numeric value, while the X-Axis (bottom labels) are showing a time-series or comparison category. You can include one or more data sets to compare, which would render as multiple lines in the chart.
+
+Learn more about our [line charts](chart-types-line.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Doughnut Chart
+
+The $PlatformShort$ Doughnut Chart, or Doughnut Graph, is a variant of a Pie Chart, proportionally illustrating the occurrences of a variable in a circle to represents parts of a whole. The doughnut chart has a circular opening at the center of the pie chart, where a title or category explanation can be displayed. Doughnut charts can support multiple concentric rings, with built-in support for visualizing hierarchical data.
+
+Learn more about our [doughnut charts](../doughnut-chart.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### $PlatformShort$ Area Chart
+
+Learn more about our [area charts](chart-types-area.md)
+
+<!-- <div class="sample-container loading" style="height: 500px">
+    <iframe></iframe>
+</div>
+<div class="divider--half"></div> -->
+
+### Sparkline Chart 
+
+A Sparkline Chart, or Sparkline Graph is a type of category graph intended for rendering within a small-scale layout such as within a grid cell, or anywhere a word-sized visualization is needed to tell a data story. Like other $PlatformShort$ chart types, the Sparkline Chart has several visual elements and corresponding features that can be configured and customized such as the chart type, markers, ranges, trendlines, unknown value plotting, and tooltips. Sparkline charts can render as a Line Chart, Area Chart, Column Chart or Win / Loss Chart. The difference between the full-sized chart equivalent to the Spark-chart, is the Y-Axis (left side labels) and X-Axis (bottom labels) are not visible. Learn more about our [sparkline charts](../sparkline.md).
+
```

Comment:
I think we have a good example for the Sparkline and we can still use it.

https://www.infragistics.com/products/ignite-ui-angular/angular/components/sparkline

Reply:
This is far too complicated showing how a sparkling is used in a templatable grid cell. I'd go with something more atomic and standalone.

Reply:
Then we can use the first example from the link of the Sparkline topic.